src/tools/tidy/src/main.rs

```diff
@@ -82,12 +82,9 @@ fn filter_dirs(path: &Path) -> bool {
         "src/llvm",
         "src/libbacktrace",
         "src/compiler-rt",
-        "src/rt/hoedown",
         "src/rustllvm",
         "src/rust-installer",
         "src/liblibc",
-        "src/tools/cargo",
-        "src/vendor",
```

Comment:
In fact several issues are preventing me from completing `x.py test distcheck` locally... Just blindly reverted the last line